app/jobs/serialisation_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SerialisationJob < ApplicationJob
+  queue_as :serialisations
+
+  def perform(contribution_id)
+    contribution = Contribution.find(contribution_id)
+    serialisation = contribution.serialisations.rdfxml.first
+    serialisation ||= Serialisation.new(format: 'rdfxml', contribution: contribution)
+    serialisation.data = contribution.ore_aggregation.to_rdfxml
```

Comment:
While it's not possible(or unlikely, since it can only be done manually) currently, but once deletion is functional, I think this is throwing errors when the contribution has been 'wiped' as there isn't an ore_aggregation.